src/LayerPlan.cpp

```diff
@@ -1644,10 +1646,16 @@ void LayerPlan::writeGCode(GCodeExport& gcode)
                     // Prevent the final travel(s) from resetting to the 'previous' layer height.
                     gcode.setZ(final_travel_z);
                 }
-                for(unsigned int point_idx = 0; point_idx < path.points.size(); point_idx++)
+                for(unsigned int point_idx = 0; point_idx < path.points.size() - 1; point_idx++)
```

Comment:
why not use:
```c++
for(const auto& point : path.points)
{
   gcode.writeTravel(point, speed)
}
```
untested and maybe use the actual type instead of auto but we should no longer use old style loops imo

Reply:
Because it reaches up to the `path.points.size() - 1` point.

Reply:
@konskarm Good catch and stupid suggestion of me. But it so makes me wish that we add the range-v3 library (which we now can). So that we can do this save, modern and lazy. https://ericniebler.github.io/range-v3/ 
I wished we used the header only range-v3 library that would make this a breach:
```c++
for(const auto& point : path.points| ranges::views::take(path.points.size() - 1))
{
   gcode.writeTravel(point, speed)
}
```